selfdrive/controls/radard.py

```diff
@@ -134,7 +129,7 @@ def update(self, sm, rr):
 
       # create the track if it doesn't exist or it's a new track
       if ids not in self.tracks:
-        self.tracks[ids] = Track(v_lead, self.kalman_params, 0.81 if self.radar_acc_tssp else 0.9)
+        self.tracks[ids] = Track(v_lead, self.kalman_params, math.pow(0.9, self.radar_ts / 0.05))
```

Comment:
Hi @sshane, I'd like do this, as this radar report at a lower frequency. The aLead TAU will be less sensitive. But this will fail the replay tests.
We could use fixed 0.9 for others, and 0.81 for this one. What's your suggestion? Thanks.